From: Shiji Yang <yangshiji66@outlook.com>
Date: Fri, 28 Mar 2025 21:02:27 +0800
Subject: [PATCH] ath10k-ct: silence noisy log caused by flushing queue

.flush() is a regular mac80211 operation aims to clear all pending
frames from the hardware queue. Only developers need to care about it.

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
---
 ath10k-6.14/mac.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/ath10k-6.10/mac.c
+++ b/ath10k-6.10/mac.c
@@ -9061,12 +9061,12 @@ static void ath10k_flush(struct ieee8021
         if (vif) {
                 arvif = (void *)vif->drv_priv;
                 vid = arvif->vdev_id;
-                ath10k_info(ar, "mac flush vdev %d drop %d queues 0x%x ar->paused: 0x%lx  arvif->paused: 0x%lx\n",
-                            arvif->vdev_id, drop, queues, ar->tx_paused, arvif->tx_paused);
+                ath10k_dbg(ar, ATH10K_DBG_MAC, "mac flush vdev %d drop %d queues 0x%x ar->paused: 0x%lx  arvif->paused: 0x%lx\n",
+                           arvif->vdev_id, drop, queues, ar->tx_paused, arvif->tx_paused);
         }
         else {
-                ath10k_info(ar, "mac flush null vif, drop %d queues 0x%x\n",
-                            drop, queues);
+                ath10k_dbg(ar, ATH10K_DBG_MAC, "mac flush null vif, drop %d queues 0x%x\n",
+                           drop, queues);
         }
 
 
